backend/src/main/java/com/soen341/instagram/model/Comment.java

```diff
@@ -1,12 +1,15 @@
 package com.soen341.instagram.model;
```

Comment:
Overall comment, we need to be careful when merging. Luc's recent pull request changes some things such as using TimeStamp instead of Time for the Date created, we want to avoid conflicts since the file is being changed by both of you